packages/gatsby-transformer-sharp/src/customize-schema.js

```diff
@@ -375,6 +380,182 @@ const fluidNodeType = ({
+        type: GraphQLInt,
+      },
+      placeholder: {
+        type: ImagePlaceholderType,
+        defaultValue: `blurred`,
+        description: stripIndent`
+        Format of generated placeholder image. 
+        DOMINANT_COLOR: a solid color, calculated from the dominant color of the image. 
+        BASE64: a blurred, low resolution image, encoded as a base64 data URI
+        TRACED_SVG: a low-resolution traced SVG of the image.
+        NONE: no placeholder. Set "background" to use a fixed background color.`,
+      },
+      tracedSVGOptions: {
+        type: PotraceType,
+        defaultValue: false,
+        description: `Options for traced placeholder SVGs. You also should set placeholder to SVG.`,
+      },
+      webP: {
+        type: GraphQLBoolean,
+        defaultValue: true,
+        description: `Generate images in WebP format as well as matching the input format. This is the default (and strongly recommended), but will add to processing time.`,
+      },
+      outputPixelDensities: {
+        type: GraphQLList(GraphQLInt),
+        description: stripIndent`
+        A list of image pixel densities to generate, for high-resolution (retina) screens. It will never generate images larger than the source, and will always a 1x image. 
+        Default is [ 1, 2 ] for fixed images, meaning 1x, 2x, 3x, and [0.25, 0.5, 1, 2] for fluid. In this case, an image with a fluid layout and width = 400 would generate images at 100, 200, 400 and 800px wide`,
+      },
+      sizes: {
+        type: GraphQLString,
+        defaultValue: ``,
+        description: stripIndent`
+        The "sizes" property, passed to the img tag. This describes the display size of the image. 
+        This does not affect the generated images, but is used by the browser to decide which images to download. You can leave this blank for fixed images, or if the responsive image
+        container will be the full width of the screen. In these cases we will generate an appropriate value.
+        `,
+      },
+      base64Width: {
+        type: GraphQLInt,
+      },
+      grayscale: {
+        type: GraphQLBoolean,
+        defaultValue: false,
+      },
+      jpegProgressive: {
+        type: GraphQLBoolean,
+        defaultValue: true,
+      },
+      pngCompressionSpeed: {
+        type: GraphQLInt,
+        defaultValue: DEFAULT_PNG_COMPRESSION_SPEED,
+      },
+      duotone: {
+        type: DuotoneGradientType,
+        defaultValue: false,
+      },
+      quality: {
+        type: GraphQLInt,
+      },
+      jpegQuality: {
+        type: GraphQLInt,
+      },
+      pngQuality: {
+        type: GraphQLInt,
+      },
+      webpQuality: {
+        type: GraphQLInt,
+      },
+      toFormat: {
+        type: ImageFormatType,
+        defaultValue: ``,
+      },
+      toFormatBase64: {
+        type: ImageFormatType,
+        defaultValue: ``,
+        description: `Force output format. Default is to use the same as the input format`,
+      },
```

Comment:
Unsure if this makes sense into gatsbyProps

Reply:
Yeah, maybe in an options object for the blurred preview